src/pages/trending.tsx

```diff
@@ -0,0 +1,284 @@
+            <div className="bg-slate-50 inline-flex rounded-full p-4 mb-4">
+              <Search className="w-8 h-8 text-slate-400" />
+            </div>
+            <h3 className="text-xl font-medium text-slate-900 mb-2">
+              No Matching Snippets
+            </h3>
+            <p className="text-slate-500 max-w-md mx-auto mb-6">
+              {searchQuery
+                ? `No snippets match your search for "${searchQuery}".`
+                : category !== "all"
+                  ? `No ${category} snippets found in the trending list.`
+                  : "There are no trending snippets at the moment."}
+            </p>
+          </div>
+        ) : (
+          <div className="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 gap-4">
+            {filteredSnippets
+              ?.sort((a, b) => b.updated_at.localeCompare(a.updated_at))
+              ?.map((snippet) => (
+                <Link
+                  key={snippet.snippet_id}
+                  href={`/${snippet.owner_name}/${snippet.unscoped_name}`}
+                >
+                  <div className="border p-4 rounded-md hover:shadow-md transition-shadow flex flex-col gap-4">
+                    <div className="flex items-start gap-4">
+                      <div className="h-16 w-16 flex-shrink-0 rounded-md overflow-hidden">
+                        <OptimizedImage
+                          src={`${apiBaseUrl}/snippets/images/${snippet.owner_name}/${snippet.unscoped_name}/pcb.svg`}
+                          alt={`${snippet.owner_name}'s profile`}
+                          className="object-cover h-full w-full transition-transform duration-300 -rotate-45 hover:rotate-0 hover:scale-110 scale-150"
+                        />
+                      </div>
+                      <div className="flex-1 min-w-0">
+                        <div className="flex justify-between items-start mb-[2px] -mt-[3px]">
+                          <h2 className="text-md font-semibold truncate pr-[30px]">
+                            <span className="text-gray-700 text-md">
+                              {snippet.owner_name}
+                            </span>
+                            <span className="mx-1">/</span>
+                            <span className="text-gray-900">
+                              {snippet.unscoped_name}
+                            </span>
+                          </h2>
+                          <div className="flex items-center gap-2">
+                            <SnippetTypeIcon
+                              type={snippet.snippet_type}
+                              className="pt-[2.5px]"
+                            />
+                            <div className="flex items-center gap-1 text-gray-600">
+                              <StarIcon className="w-4 h-4 pt-[2.5px]" />
+                              <span className="text-[16px]">
+                                {snippet.star_count || 0}
+                              </span>
```

Comment:
shouldn't we reuse the snippet card from the other page? not a huge deal but slightly more work to maintain